client/components/Assignments/AssignmentItem/fields/DescriptionText.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {get} from 'lodash';
+
+interface IProps {
+    assignment: any;
+}
+
+export const DescriptionTextComponent = ({assignment}: IProps) => {
+    const descriptionText = get(assignment, 'description_text');
+
+    return (
+        <span className="sd-overflow-ellipsis sd-list-item--element-grow">
```

Comment:
There is a UI glitch here if the description is too long then the slugline spans multiple lines inside the first row.

In the current codebase, there is a span element encasing the `slugline` and `decription_text` which has these two classes. This enables the ellipsis across the row.

Play around and make sure long descriptions/names/sluglines places an ellipsis and does not increase the height of the first line (maybe using the two classes `sd-overflow-ellipsis sd-list-item--element-grow` on the first row in `renderContentColumn`)

Reply:
Good to know, I'll test it with long descriptions 👍